content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -471,36 +470,35 @@ <h2 id="toc-webaudio-api">Using getUserMedia with the Web Audio API</h2>
+<p>Historically, device access on the web has been a tough nut to crack. <a href="https://www.slideshare.net/jamesgpearce/mobile-device-apis">Many tried, few succeeded</a>. Most of the early ideas never gained widespread adoption or took hold outside of a proprietary environment. Perhaps the main problem has been that the web's security model is <em>very</em> different from the native world. In particular, you probably don't want every Joe Shmoe website to have random access to your video camera or microphone. It's been difficult to get right.</p>
 
-<p>Since then, driven by the increasingly ubiquitous capabilities of mobile devices, the web has begun to provide much richer functionality. We now have APIs to <a href="https://developers.google.com/web/updates/2016/12/imagecapture" title="Web Fundamentals article about the Image Capture API">take photos and control camera settings</a>, <a href="https://developers.google.com/web/updates/2016/01/mediarecorder" title="Web Fundamentals article: Record Audio and Video with MediaRecorder">record audio and video</a>, and to access other types of sensor data such as location, motion and device orientation. The <a href="https://developers.google.com/web/updates/2017/09/sensors-for-the-web" title="What is Generic Sensor API?">Generic Sensor framework</a> ties all this together, alongside generic APIs to enable web applications to <a href="https://developers.google.com/web/updates/2016/03/access-usb-devices-on-the-web" title="Access USB Devices on the Web">access USB</a> and <a href="https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web" title="Interact with Bluetooth devices on the Web">interact with Bluetooth</a> devices.</p>
+<p>Since then, driven by the increasingly ubiquitous capabilities of mobile devices, the web has begun to provide much richer functionality. You now have APIs to <a href="https://developers.google.com/web/updates/2016/12/imagecapture" title="Web Fundamentals article about the Image Capture API">take photos and control camera settings</a>, <a href="https://developers.google.com/web/updates/2016/01/mediarecorder" title="Web Fundamentals article: Record Audio and Video with MediaRecorder">record audio and video</a>, and access other types of sensor data, such as location, motion, and device orientation. The <a href="https://developers.google.com/web/updates/2017/09/sensors-for-the-web" title="What is Generic Sensor API?">Generic Sensor framework</a> ties all this together, alongside generic APIs to enable web apps to <a href="https://developers.google.com/web/updates/2016/03/access-usb-devices-on-the-web" title="Access USB Devices on the Web">access USB</a> and <a href="https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web" title="Interact with Bluetooth devices on the Web">interact with Bluetooth</a> devices.</p>
```

Comment:
Use web.dev/usb and web.dev/bluetooth for `to access USB and interact with Bluetooth devices` links.

Reply:
Done.